docs/compiling-a-contract.rst

```diff
@@ -29,7 +31,7 @@ Include the ``-f`` flag to specify which output formats to return. Use ``vyper -
 
 .. code:: shell
 
-    $ vyper -f abi,abi_python,bytecode,bytecode_runtime,interface,external_interface,ast,annotated_ast,ir,ir_json,ir_runtime,hex-ir,asm,opcodes,opcodes_runtime,source_map,method_identifiers,userdoc,devdoc,metadata,combined_json,layout yourFileName.vy
+    $ vyper -f abi,abi_python,bytecode,bytecode_runtime,interface,external_interface,ast,annotated_ast,integrity,ir,ir_json,ir_runtime,hex-ir,asm,opcodes,opcodes_runtime,source_map,method_identifiers,userdoc,devdoc,metadata,combined_json,layout yourFileName.vy
```

Comment:
Missing flags in the docs:

- `blueprint_bytecode`
- `source_map_runtime`
- `archive`
- `solc_json`

`integrity` and `hex-ir` are missing in the cli help:

![image](https://github.com/vyperlang/vyper/assets/25297591/d9bf877c-19b8-4a93-8f1d-43092d48d127)

Reply:
hex-ir is in the help

Reply:
14f0329339f455c719a56d204ae9b31118b689c5

Reply:
> hex-ir is in the help

oh in that case you need to remove it here in the docs since `-f` flag is not required